test/recipes/95-test_external_tlsfuzzer.t

```diff
@@ -0,0 +1,28 @@
+#! /usr/bin/env perl
+# Copyright 2015-2021 The OpenSSL Project Authors. All Rights Reserved.
+#
+# Licensed under the Apache License 2.0 (the "License").  You may not use
+# this file except in compliance with the License.  You can obtain a copy
+# in the file LICENSE in the source distribution or at
+# https://www.openssl.org/source/license.html
+
+
+use OpenSSL::Test;
+use OpenSSL::Test::Utils;
+use OpenSSL::Test qw/:DEFAULT data_file data_dir bldtop_dir srctop_dir cmdstr/;
+use Cwd qw(abs_path);
+
+setup("test_external_tlsfuzzer");
+
+plan skip_all => "No external tests in this configuration"
+    if disabled("external-tests");
+plan skip_all => "TLSFuzzer tests not available on Windows or VMS"
+    if $^O =~ /^(VMS|MSWin32)$/;
+plan skip_all => "TLSFuzzer tests not supported in out of tree builds"
+    if bldtop_dir() ne srctop_dir();
```

Comment:
Perhaps the test should be also skipped if the tlsfuzzer submodule is not present. Similarly to how some other external test check presence of the external sources.

Reply:
Not sure - we have submodules to ensure the presence of the sources, don't we?

Reply:
Yes, but you might want to run just some of the external tests. Of course you can always selectively run the individual tests, but... But yeah the gost external test recipe does not check for the presence of the gost sources either.